### PR TITLE
Update 11-using-arrays-transformations.md

### PR DESCRIPTION
**Pull request by DebTho**
I suggested a clarification that we will be skipping to the second numbered list of instructions. Reason being the two below look quite similar at first glance when you are busy with teaching:
2. On the Authors column...
1. On the Authors column...

<details>
<summary><strong>Instructions</strong></summary>

Thanks for contributing! :heart:

If this contribution is for instructor training, please email the link to this contribution to
instructor.training@carpentries.org so we can record your progress. You've completed your contribution
step for instructor checkout by submitting this contribution!

Keep in mind that **lesson maintainers are volunteers** and it may take them some time to
respond to your contribution. Although not all contributions can be incorporated into the lesson
materials, we appreciate your time and effort to improve the curriculum. If you have any questions
about the lesson maintenance process or would like to volunteer your time as a contribution
reviewer, please contact The Carpentries Team at team@carpentries.org.

You may delete these instructions from your comment.

\- The Carpentries
</details>

### CODE REVIEW

**Comment by jas58** [comment]
Thank you, @DebTho for making this suggestion. We'll make time to review it next week.
Welcome to the Carpentries!

**Comment by jas58** [comment]
@ostephens 
I wonder if there's a different ordered sequence we could use between the three lists?
1-7
i-iii
a-b
Then the opening statement could specify ii. On the Authors

I can see why the second appearance of On the Authors is assumed (because it's the only other occurrence without a comma before use) but I agree that it is confusing.  

@zkamvar Is it possible (especially moving to the markdown of Workbench in a few months) to change the ordered list display choice?

**Comment by ostephens** [comment]
I'd be very happy with different numbering schemes (or point numbering like 1.1, 1.2, 2.1, 2.2 etc.) - however I don't think markdown is great at dealing with numbered lists beyond the basic 1,2,3,4. But if it can be done that would be great! Otherwise I'm also happy with the clarification text that @DebTho has added in this PR

**Comment by DebTho** [comment]
Hi, I think either solution would be fine and very helpful with providing clarification - which will be especially useful for finding the right instruction when instructors are busy with training!

**Review by jas58 — COMMENTED** [review]
Agree with @ostephens to add this clarifying text.

**Comment by jas58** [comment]
Thank you, @DebTho for helping improve the Carpentries!

**Comment by zkamvar** [comment]
> @ostephens I wonder if there's a different ordered sequence we could use between the three lists? 1-7 i-iii a-b Then the opening statement could specify ii. On the Authors
> 
> I can see why the second appearance of On the Authors is assumed (because it's the only other occurrence without a comma before use) but I agree that it is confusing.
> 
> @zkamvar Is it possible (especially moving to the markdown of Workbench in a few months) to change the ordered list display choice?

Yes, it is possible to change the ordered list display choice by using different IDs.

[Here's an example](https://pandoc.org/try/?params=%7B%22text%22%3A%221.+one%5Cn2.+two%5Cn3.+tree%5Cn%5Cni.+one%5Cnii.+two%5Cniii.+three%5Cn%5Cna.+one%5Cnb.+two%5Cnc.+three%22%2C%22to%22%3A%22html5%22%2C%22from%22%3A%22markdown%22%2C%22standalone%22%3Afalse%2C%22embed-resources%22%3Afalse%2C%22table-of-contents%22%3Afalse%2C%22number-sections%22%3Afalse%2C%22citeproc%22%3Afalse%2C%22html-math-method%22%3A%22plain%22%2C%22wrap%22%3A%22auto%22%2C%22highlight-style%22%3Anull%2C%22files%22%3A%7B%7D%2C%22template%22%3Anull%7D) of how the following text renders in pandoc:


```markdown
1. one
2. two
3. tree

i. one
ii. two
iii. three

a. one
b. two
c. three
```

```html
<ol type="1">
<li>one</li>
<li>two</li>
<li>tree</li>
</ol>
<ol type="i">
<li>one</li>
<li>two</li>
<li>three</li>
</ol>
<ol type="a">
<li>one</li>
<li>two</li>
<li>three</li>
</ol>
```

**Comment by DebTho** [comment]
Hi there, I was teaching this yesterday and noticed that each item in the ordered sequence is not on a separate line. Could someone help by putting each on a new line as it's now quite hard to follow. @jas58 ? Thanks!

**Comment by ostephens** [comment]
@DebTho @jas58 Looks like the approach we took isn't effective with the markdown so I've consolidated the steps into a single numbered list and updated the reference that originated this PR to reference the appropriate line.

Because this was affecting the teaching of the lesson I've gone ahead and merged this straightaway, but happy to look again - just wanted a quick fix to remove the readability issue.

Take a look at https://librarycarpentry.org/lc-open-refine/11-using-arrays-transformations/index.html and let me know if you think this is OK
